frontend/src/metabase/visualizations/components/legend/LegendItem.styled.tsx

```diff
@@ -16,7 +16,7 @@ export const LegendItemRoot = styled.div<{ isVertical: boolean }>`
 
 export const LegendItemLabel = styled.div<{ isMuted: boolean }>`
   display: flex;
-  align-items: baseline;
+  align-items: center;
```

Comment:
Wasn't sure if there was a good reason these were `baseline` but it was causing some weird alignment issues.
![Screenshot 2023-12-20 at 11 06 00 AM](https://github.com/metabase/metabase/assets/5248953/3369698d-bf33-42fe-b065-da1c2f1a4391)